.stylelintrc

```diff
@@ -4,7 +4,6 @@
     "stylelint-config-sass-guidelines",
     "stylelint-config-prettier"
```

Comment:
```suggestion
    "stylelint-config-standard-scss",
    "stylelint-config-sass-guidelines",
    "stylelint-config-prettier"
```

Reply:
I think you need to add the `stylelint-config-standard-scss` to the config file too.